modules/core/src/main/scala/com.fortysevendeg.hood/csv/CsvService.scala

```diff
@@ -18,16 +18,15 @@ package com.fortysevendeg.hood.csv
 
```

Comment:
Same here, we are using a single folder called com.fortysevendeg.hood instead of com/fortysevendeg/hood

Reply:
Same as above